flink-streaming-java/src/main/java/org/apache/flink/streaming/api/environment/StreamExecutionEnvironment.java

```diff
@@ -995,8 +993,7 @@ public TimeCharacteristic getStreamTimeCharacteristic() {
 	public <OUT> DataStreamSource<OUT> readFile(FileInputFormat<OUT> inputFormat,
 												String filePath,
 												FileProcessingMode watchType,
-												long interval,
-												FilePathFilter filter) {
+												long interval) {
```

Comment:
This is a breaking API change. `StreamExecutionEnvironment` is annotated`@Public`.